ssclj/rpm/src/main/systemd/ssclj.service

```diff
@@ -1,7 +1,8 @@
 [Unit]
 Description=SlipStream API
-After=syslog.target network.target
-After=hsqldb.service elasticsearch.service
+After=syslog.target
+After=network.target
+After=hsqldb.service
```

Comment:
@st If I recall correctly, hsqldb is no longer needed by ssclj.  I think, the `After` should read like this

```
-After=syslog.target network.target
 -After=elasticsearch.service
```
Please check.